readme.txt

```diff
@@ -57,7 +57,7 @@ choose date as your type for your field on your widget too.
 
 == Changelog ==
 
-= 2.13 =
+= 2.13.1 =
```

Comment:
Don't rename the old changelog item, add a new item

Reply:
Done.

= 2.14 =
* Error fixes for block widget editor
* Two new fields "Form name" and "Form ID" are added